queries.sql

```diff
@@ -49,4 +49,33 @@ GROUP BY species;
 
 SELECT species, AVG(escape_attempts) FROM animals
 WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
```

Comment:
- [ ] One of the requirements is that you include the snapshot of queries in your pull request summary. The snapshot for the result of this query is incorrect. Kindly include the correct snapshot. 👍